lucida/commandcenter/Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 	rm -rf lucidaservice lucidatypes
 
 start_server:
-	python app.py
+  source ${PWD}/../../tools/python_2_7_12/bin/activate && python app.py
```

Comment:
@KamalGalrani, I'm getting an error here because there are only 2 spaces before source. Can you fix this?